Resources/Locale/en-US/strip/strippable-component.ftl

```diff
@@ -6,13 +6,13 @@ strippable-component-cannot-put-message = {$owner} cannot put that there!
 strippable-component-item-slot-free-message = {$owner} has nothing there!
 strippable-component-cannot-unequip-message = {$owner} cannot unequip that!
 strippable-component-cannot-drop-message = {$owner} cannot drop that!
-strippable-component-alert-owner = {$user} is removing your {$item}!
+strippable-component-alert-owner = {$user} { CONJUGATE-BE($user) } removing your {$item}!
 strippable-component-alert-owner-hidden = You feel someone fumbling in your {$slot}!
-strippable-component-alert-owner-insert = {$user} is putting {$item} on you!
-strippable-component-alert-owner-insert-hand = {$user} is putting {$item} in your hand!
+strippable-component-alert-owner-insert = {$user} { CONJUGATE-BE($user) } putting {$item} on you!
+strippable-component-alert-owner-insert-hand = {$user} { CONJUGATE-BE($user) } putting {$item} in your hand!
 
 # generic warning for when a user interacts with your equipped items.
-strippable-component-alert-owner-interact = {$user} is fumbling around with your {$item}!
+strippable-component-alert-owner-interact = {$user} { CONJUGATE-BE($user) } fumbling around with your {$item}!
```

Comment:
All of this should be "is" as they sound better first person then "are"

The insert interactions should have articles before them if possible (The, An, A)
(User is putting an/a/the item on you)